airflow/providers/google/cloud/hooks/automl.py

```diff
@@ -29,10 +29,10 @@
     ImageObjectDetectionModelDeploymentMetadata, InputConfig, Model, Operation, PredictResponse, TableSpec,
 )
 
-from airflow.providers.google.cloud.hooks.base import CloudBaseHook
+from airflow.providers.google.common.hooks.base import GoogleBaseHook
```

Comment:
```suggestion
from airflow.providers.google.common.hooks.base_google import GoogleBaseHook
```
We agreed to such a convention in the case of AWS. In the case of Google, we should also use it.  I will add a commit to this branch, which corrects it, and then from this branch I will start new PR, which will correct other errors.